runtime/buildtools.mk

```diff
@@ -189,7 +189,10 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
 configure : constantpool nls
 	mkdir -p build && cd build && $(CMAKE) -C ../cmake/caches/$(SPEC).cmake  $(CMAKE_ARGS) $(EXTRA_CMAKE_ARGS) ..
 else
-configure : uma
+j9includegen:
```

Comment:
`j9includegen` must depend upon `uma` else the necessary makefiles will not have been generated.
Also the option `-f makefile` is redundant.

Reply:
`j9includegen` should also be added to the list of `.PHONY` targets.